homeassistant/components/light/osramlightify.py

```diff
@@ -50,20 +57,26 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
                 host, str(err))
             _LOGGER.exception(msg)
             return False
-        setup_bridge(bridge, add_devices)
+        setup_bridge(bridge, add_devices, add_groups)
     else:
         _LOGGER.error("No host found in configuration")
         return False
 
 
-def setup_bridge(bridge, add_devices_callback):
-    """Set up the Lightify bridge."""
+def setup_bridge(bridge, add_devices_callback, add_groups):
+    """Setup the Lightify bridge."""
```

Comment:
If I remove it, the `DEFAULT_ALLOW_LIGHTIFY_GROUPS = True` config becomes useless.